hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotPurgeRequest.java

```diff
@@ -101,9 +101,11 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, Execut
           continue;
         }
         SnapshotInfo nextSnapshot = SnapshotUtils.getNextSnapshot(ozoneManager, snapshotChainManager, fromSnapshot);
-
-        // Step 1: Update the deep clean flag for the next active snapshot
+        SnapshotInfo nextToNextSnapshot = SnapshotUtils.getNextSnapshot(ozoneManager, snapshotChainManager,
```

Comment:
nextToNextSnapshot can be null. But this case is handled in updateSnapshotInfoAndCache()